attendance/lib/ProfilePage/profile_page_view.dart

```diff
@@ -0,0 +1,222 @@
+                                    ),
+                                    onPressed: () =>
+                                        popupEditDialog(
+                                            nativeNameController,
+                                            20,
+                                            InputType.TEXT,
+                                            TextDirection.RTL),
+                                    shape: RoundedRectangleBorder(
+                                        borderRadius:
+                                        BorderRadius.circular(20.0)))),
+                            SizedBox(height: 25.0),
+                            Container(
+                                margin: EdgeInsets.only(left: 40.0),
+                                height: 30.0,
+                                width: 70.0,
+                                child: OutlineButton(
+                                    splashColor: Colors.lightBlueAccent,
+                                    borderSide:
+                                    BorderSide(color: Colors.lightBlueAccent),
+                                    child: Text(
+                                      "Edit",
+                                      style:
+                                      TextStyle(color: Colors.lightBlueAccent),
+                                    ),
+                                    onPressed: () =>
+                                        popupEditDialog(
+                                            phoneNumberController,
+                                            11,
+                                            InputType.NUMBER,
+                                            TextDirection.LTR),
+                                    shape: RoundedRectangleBorder(
+                                        borderRadius:
+                                        BorderRadius.circular(20.0)))),
+                          ],
+                        )
+                      ],
+                    )
+                  ],
+                ))
+          ],
+        ));
+  }
+
+  void popupEditDialog(TextEditingController controller, int max,
+      InputType type, TextDirection dir) {
+    showDialog(
+        context: context,
+        builder: (BuildContext context) => Dialog(
+            child: Container(
+              height: 150.0,
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.center,
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: <Widget>[
+                  Container(
+                    width: 250,
+                    margin: EdgeInsets.only(top: 10.0),
+                    child: Theme(
+                      data: ThemeData(hintColor: Colors.lightBlueAccent),
+                      child: TextField(
+                        controller: controller,
+                        onChanged: (text) {
+                          //TODO: Update..
```

Comment:
@Bahaaib This todo is not a puzzle format

Reply:
Oops. I added it days ago before knowing about pdd.